src/main/java/org/jmhsrobotics/frc2024/controlBoard/CompControl.java

```diff
@@ -24,7 +24,7 @@ private boolean isTurbo() {
 
 	@Override
 	public double xInput() {
-		return this.driver.getLeftX() * (isTurbo() ? 1.0 : slowMode);
+		return this.driver.getLeftX() * (isTurbo() ? 1.0 : slowMode) * 0.7;
 	}
 
```

Comment:
I think this change was meant to be limiting the rotation speed not the x speed. With this change it makes X rotation 30% slower. That explains why the driver was not seeing an effect last practice.